doc/other_tools.rst

```diff
@@ -66,5 +66,7 @@ that works for more grid types.
 - `Regridder in Iris <https://scitools-iris.readthedocs.io/en/v3.4.1/userguide/interpolation_and_regridding.html>`_
   (*Python package*)
 
-- `Regridder in UV-CDAT <https://uvcdat.llnl.gov/documentation/cdms/cdms_4.html>`_
+- `Regridder in xCDAT <https://xcdat.readthedocs.io/en/latest/>`_
```

Comment:
```suggestion
- `Regridder in xCDAT <https://xcdat.readthedocs.io/en/latest/generated/xcdat.regridder.accessor.RegridderAccessor.html>`_
```

I updated the link to point to the regridding accessor instead of the homepage.